tsconfig-aot.json

```diff
@@ -13,7 +13,12 @@
     "noEmitHelpers": true,
     "rootDir": ".",
     "noUnusedLocals": true,
-    "noUnusedParameters": true
+    "noUnusedParameters": true,
+    "types": [
```

Comment:
Let me double check that. I think I needed this for AT and TS IDE plugins to pick up typings.